pkg/registry/configurator.go

```diff
@@ -52,7 +52,8 @@ func NewConfigurator(deployHosts []net.IP,
 	containerRuntimeInfo containerruntime.Info,
 	regConfig v2.Registry,
 	infraDriver infradriver.InfraDriver,
-	distributor imagedistributor.Distributor) (Configurator, error) {
+	distributor imagedistributor.Distributor,
```

Comment:
go fmt code


Generally, in code style, we don't use ) as the first character of a line

Reply:
Oops sorry, I was mainly using Rust which encourages this style. Fixed.